default.nix

```diff
@@ -0,0 +1,11 @@
+{ mkDerivation, base, gauge, stdenv }:
```

Comment:
Are these nix files necessary? Since we don't check `nix` bulds on CI, let's not have these files not to give a false impression that we support nix builds robustly :slightly_smiling_face: